docs/secure-connections/sauce-connect/advanced/specifications.md

```diff
@@ -0,0 +1,61 @@
+---
+id: specifications
+title: Sauce Connect Proxy™ Specifications
```

Comment:
Legally, we only need to apply the trademark symbol on the [first prominent mention in our docs](https://docs.saucelabs.com/secure-connections/).

```suggestion
title: Sauce Connect Proxy Specifications
```